vspace/src/main/webapp/WEB-INF/views/exhibition/module.jsp

```diff
@@ -1,12 +1,165 @@
+									</c:forEach></li>
+							</ul>
+						</nav>
+					</div>
+					<div id="slide_content">
+						<nav class="slide_navbar navbar-default">
+							<div class="container-fluid">
+								<div class="navbar-header">
+									<button type="button" id="slide_sidebarCollapse"
+										class="btn btn-info">
+										<i class="fas fa-align-justify"></i>
+									</button>
+								</div>
+							</div>
+						</nav>
+						<div class="Group_8_Class">
+							<c:forEach items="${currentSlideCon.contents}" var="contents">
+								<c:if test="${contents['class'].simpleName ==  'ImageBlock'}">
+									<div style="margin: 1%;" class="valueDiv" id="${contents.id}">
+										<img id="${contents.id}" class="imgDiv" style="margin: 1%;"
+											src="<c:url value="/api/image/${contents.image.id}" />" />
+									</div>
+								</c:if>
+								<c:if test="${contents['class'].simpleName ==  'TextBlock'}">
+									<div id="${contents.id}" class="textDiv"
+										style="margin: 10px; text-align: justify; padding-right: 250px">
+										<p>${contents.text}</p>
+									</div>
+								</c:if>
+							</c:forEach>
+						</div>
+						<div class="Group_7_Class">
+							<c:if test="${prevSlide !=  ''}">
+								<a
+									href="/vspace/exhibit/module/${module.id}/sequence/${currentSequenceId}/slide/${prevSlide}">
+									<div class="Slideshow_previous Slideshow_previous_Class">
+										<svg class="Ellipse_11">
+									<ellipse fill="rgba(255,255,255,1)" class="Ellipse_11_Class"
+												rx="36" ry="36" cx="36" cy="36">
+									</ellipse>
+								</svg>
+										<svg class="Icon_ionic_ios_arrow_back"
+											viewBox="11.251 6.194 20.021 35.021">
+									<path fill="rgba(101,101,101,1)"
+												class="Icon_ionic_ios_arrow_back_Class"
+												d="M 17.28610038757324 23.69940376281738 L 30.53693389892578 10.45899486541748 C 31.51693534851074 9.478996276855469 31.51693534851074 7.894316673278809 30.53693389892578 6.92474365234375 C 29.55693244934082 5.944746017456055 27.97225761413574 5.95517110824585 26.99225997924805 6.92474365234375 L 11.9795093536377 21.92706680297852 C 11.03078651428223 22.87578582763672 11.00993728637695 24.39791297912598 11.90652942657471 25.3779125213623 L 26.98183441162109 40.48448944091797 C 27.47183418273926 40.97449111938477 28.11821746826172 41.21427917480469 28.75417137145996 41.21427917480469 C 29.39012908935547 41.21427917480469 30.0365104675293 40.97449111938477 30.52651405334473 40.48448944091797 C 31.50650978088379 39.50448989868164 31.50650978088379 37.91981506347656 30.52651405334473 36.95023727416992 L 17.28610038757324 23.69940376281738 Z">
+									</path>
+								</svg>
+									</div>
+								</a>
+							</c:if>
+							<c:if test="${nextSlide !=  ''}">
+								<a
+									href="/vspace/exhibit/module/${module.id}/sequence/${currentSequenceId}/slide/${nextSlide}">
```

Comment:
this is not how you create links (this will raise a not found (see my other ocmments)).